lang_GENERIC/analyze/naming_ast.ml

```diff
@@ -0,0 +1,260 @@
+(* Yoann Padioleau
+ *
+ * Copyright (C) 2020 r2c
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file license.txt.
+ * 
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * license.txt for more details.
+ *)
+open Common
+open Ast_generic
+module Ast = Ast_generic
+module V = Visitor_ast
+
+(*****************************************************************************)
+(* Prelude *)
```

Comment:
most verbose prelude I read in a file! Woot!

Reply:
yes :) 1/4 of the time coding the feature was to write this comment. It's a bit obscure and maybe useless, but I like those prelude comment.